docker/Dockerfile-local

```diff
@@ -1,15 +1,19 @@
 FROM ruby:2.3.1
 
+RUN apt-get update && \
```

Comment:
There is a different PR for this called "docker-changes" - think it may still be under review. Cheers